[Analysis Engine] More tests cases for data export integration

Mapleia opened the pull request:
## Changes
Added more test cases to deal with similar sources, similar values, similar names etc.

## Requirements
Requires: N/A

## PR Dependencies
Depends on: #17  

## Merged Branches and Other Dependencies
Merged branch(es):
Other dependencies:

## Subtasks / Progress
- [x] Add case for health data

rlnsy commented:
@Mapleia Looks like there are a lot of linting/whitespace changes - which is fine but can we move those into an intermediate PR? That way we can look more accurately at what has changed in the test file.

rlnsy reviewed (changes requested):
See comment.

Mapleia commented:
@rlnsy I don't see those whitespace changes anymore, is it gone for you too?

rlnsy commented:
Yep, will review now

rlnsy commented:
merge conflicts

rlnsy commented:
@Mapleia Need to fix merge conflicts, then set master as base branch and potentially fix the feature branch.

noktoa committed:
Fixed branch ready to merge
Squashed commit of the following:

commit d4779aeccc234afbf382fa16e9c058f946f046ab
Merge: aad077b 9915c90
Author: Mapleia <mapleia01@gmail.com>
Date:   Fri Sep 3 10:21:49 2021 -0700

    Merge remote-tracking branch 'origin/add-test-cases' into fix

commit aad077bc20250ee00eeeb1ab66aa132daf2caba0
Merge: f00858b 4d24101
Author: Rowan Lindsay <rowan@rowanlindsay.com>
Date:   Tue Aug 24 10:26:02 2021 -0700

    Merge remote-tracking branch 'origin/master' into neogma-fixtures

commit f00858b6c8ba7c9fc7d5ef02cad69a308dddc5cd
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu Aug 19 23:31:30 2021 -0700

    conflict fix

commit 50221da18e43015e36fc196cf7c507bf81f71b6c
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu Aug 19 23:19:55 2021 -0700

    fixed build error

commit 42f3a1f9c521d12294a78296b332b5a9c63340de
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu Aug 19 22:21:42 2021 -0700

    Fixed branch ready to merge.

    Squashed commit of the following:

    commit 1638942af66e55e9216db09932e9cb207ac48eb5
    Merge: 3cccbba 4d6a114
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Aug 19 22:20:50 2021 -0700

        Merge remote-tracking branch 'origin/neogma-fixtures' into fix

    commit 3cccbba875439511c459d99adb0c9fc1137bdb74
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Aug 19 21:41:25 2021 -0700

        relative path for import

    commit 85ac61dc14f8b8b05b0539be6a8bfc5c58bb4a8f
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Aug 19 21:07:27 2021 -0700

        build fixes

    commit 52fdac4578c18cee2c88c0a0447f91c99a1e2316
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Aug 19 20:59:45 2021 -0700

        lint fix

    commit 24fcd23c0ce9906aae40d2accfe1fa0483ee3d59
    Merge: 29db18d 68cd177
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Aug 19 20:57:14 2021 -0700

        Merge branch 'neogma-tweaks' of https://github.com/bipolarbridges/data-services into neogma-tweaks

    commit 29db18de1446e1c2059bb2826fdf85c881243e5e
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Aug 19 20:53:59 2021 -0700

        match origin

    commit 527269bd25694d415ce44b41745ef2ef361b6889
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Sat Aug 14 02:36:07 2021 -0700

        Fixed branch ready to merge

    commit 68cd177e608ea07dd03422bd11ad9b0cf480cccd
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Sat Aug 14 02:36:07 2021 -0700

        Fixed branch ready to merge

    commit 4d6a114059e44c059060fe527092b6c5b509eeea
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Aug 13 17:31:19 2021 -0700

        fixed nodemon and import

    commit 622fd868dbf5ea753609a37f712ec036ff34ea8f
    Merge: b5c46a5 95bf9ef
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Aug 13 17:26:54 2021 -0700

        Merge branch 'neogma-fixtures' of https://github.com/bipolarbridges/data-services into neogma-fixtures

    commit b5c46a5c97907ed4c0a054b0662a31602226c202
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Aug 13 15:43:51 2021 -0700

        remove redundant npm commands

    commit 95bf9ef5accb7f0f4417ec0ff45ee734f00fc071
    Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
    Date:   Sat Aug 7 19:09:43 2021 -0700

        lint fixes

    commit 1968a718a35f00496100608bc5246c2cbd9487b0
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Tue Jul 13 13:05:15 2021 -0700

        import errors

    commit 02d31c1468c1537d779e6297a492a336f2429989
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jul 5 18:02:05 2021 -0700

        lint and build fix

    commit 1fb203cb568199602ddb254eb0a08a51b5ad534b
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jul 5 17:59:00 2021 -0700

        lint fix

    commit 0a1c5858a6fabf6d48fd030f99c15111b9f42d6b
    Merge: 3de979e df1da6b
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jul 5 17:56:43 2021 -0700

        Merge branch 'neogma-fixtures' of https://github.com/bipolarbridges/data-services into neogma-fixtures from rebasing

    commit 3de979ef96f92d0bbb8c87bac2aee1f3862d7d28
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Mon Jun 21 16:12:00 2021 -0700

        clean up

    commit 3e9752c2bbdeaf32033bcfbd34efcf03c6a49a86
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jun 21 15:27:26 2021 -0700

        small type fixes, remove unneeded imports

    commit 16c4c93242f418666188560ebe4840f42a0cad24
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Mon Jun 21 00:05:47 2021 -0700

        small refactor:

    commit 443ad138da7c722919ffd84acd728f0707468f07
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 18:33:45 2021 -0700

        added test data fixtures using neogma

    commit b94e96f540e40791731e9e66e4749c3af66989ac
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 20:27:47 2021 -0700

        added examples fixtures and test data

    commit e366ea5334da032b44873c7ef393d025c7d411f0
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 20:42:15 2021 -0700

        new testing workflow

    commit 18654501803948e230c8a6df467c33f707f55309
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 18:33:45 2021 -0700

        added test data fixtures using neogma

    commit 45f7bc400f52f842ea095f73f36206c8c33868a0
    Merge: 38e0bd3 e961a61
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jul 5 17:41:28 2021 -0700

        Merge branch 'neogma-tweaks' of https://github.com/bipolarbridges/data-services into neogma-tweaks from rebasing

    commit 38e0bd3b9f241699aec2fda58f43175b3b49ed1a
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jul 5 17:36:15 2021 -0700

        typing fix

    commit b1159a695fc2e294e5d52f1889f47e606bff9226
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Fri Jun 18 17:06:28 2021 -0700

        fix the lint

    commit a4da0fe7536a2f634e7d888bcd4561a4df0e5d5c
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Fri Jun 18 17:03:36 2021 -0700

        refactor existing methods to use OGM

    commit 59f15cd6f2e2b67b2e4c1b8c4b9912e27224ba82
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Fri Jun 18 12:04:49 2021 -0700

        upgrade neogma

    commit cf78305ecb56cc6fba980a7da08f0ed7e652daa0
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Wed Jun 16 13:36:42 2021 -0700

        typing tweaks

    commit 1b3d3e0fe461ff39ca547949423f5cbed2d37cd8
    Merge: 9390813 a6ee549
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jul 5 17:16:06 2021 -0700

        Merge branch 'update-docker-usage' of https://github.com/bipolarbridges/data-services into update-docker-usage

    commit 9390813e93db2097493d1670571cfaede7407b5d
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Fri Jun 18 11:20:34 2021 -0700

        update network name usage

    commit 42c41525db781a6ee67d1fc50daf0777d6326ea7
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Fri Jun 18 11:18:44 2021 -0700

        use new docker compose syntax

    commit 905fc4f078cca05d28788c29e912b68764b92531
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 18:15:12 2021 -0700

        cleaned up comments, made initializer live in model files

    commit 705a7bcc40edfb41e6ceee4c3bd6885b598bea6d
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 17:50:27 2021 -0700

        readme change

    commit 4156f3157fc06a1df18d08f1bee7fab57e1b9208
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 17:37:54 2021 -0700

        dependency cycle fix

    commit c62e43fd9ca13c00eb853cc3411d6fa83db30542
    Merge: 768df79 955ccc7
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 17:32:57 2021 -0700

        Merge branch 'model-schema-change' into fix

    commit 768df791bf13907e0da0d95a581690c2edf4fbec
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 17:30:11 2021 -0700

        lint fix

    commit 7ab46d9f21b2104cb32d44294df4efc93e890a34
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 16:46:30 2021 -0700

        lint:fix

    commit 3bcadb25911fc82b4461372b872883addf92cb6f
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 16:35:27 2021 -0700

        more dependencies

    commit 99914db8a26e1bd68a52bce4e56fbf809549ada1
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 16:34:42 2021 -0700

        eslint fix

    commit b79f4f9f5a1bda32df3f628aa1cd1589b38ea445
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 16:34:25 2021 -0700

        yarn lock

    commit 955ccc71ea05f9354817bcb85aded4f1394a8268
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 16:08:59 2021 -0700

        no async

    commit a12ef566aaf4b299bb5e5710069a65c3b754a31a
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 15:15:32 2021 -0700

        more lint fix, yarn lock fix

    commit 0f2222c77441331b141882fac414d90f146c0b5e
    Merge: 978bafb 109c2e1
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 15:08:57 2021 -0700

        Merge branch 'neogma-installation' into model-schema-change

    commit 109c2e1ec48ce0148c515107ffc7efd87df40642
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 14:56:35 2021 -0700

        lint fix

    commit 1bf5a760695e6b37bc9d1c87558d3f1474fae023
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 13:48:36 2021 -0700

        adding missing pckgs

    commit 682645325af8188362f9934d65c13bd95fcb78fe
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 30 13:46:26 2021 -0700

        linter change

    commit 978bafbbf1c51137136a08e65dea03fb6b95f578
    Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
    Date:   Tue Jun 29 17:18:16 2021 -0700

        removed addRelationship steps

    commit 8d3aa96f14cd0ef0bc70a3c17193290015af7b4f
    Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
    Date:   Tue Jun 29 16:41:32 2021 -0700

        lint fix

    commit c6fe1aba8a53bd56fc4556c852a00492b8590c13
    Merge: 9ae1492 bc0c2ea
    Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
    Date:   Tue Jun 29 14:30:41 2021 -0700

        Merge branch 'neogma-installation' into model-schema-change

    commit 9ae149281b95b343a8ff0d7e745fbde004a021df
    Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
    Date:   Tue Jun 29 14:16:38 2021 -0700

        removed unnecessary comments

    commit b9a6b3ed3ae9a252126d449f2399256853089747
    Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
    Date:   Tue Jun 29 14:11:29 2021 -0700

        type fix

    commit d14d5e29a07a92cc5ea19fba11384f2ff4270e64
    Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
    Date:   Tue Jun 29 14:07:05 2021 -0700

        circular dependency fix

    commit 24914287391b6eb2a4a80f421d946dde75b702b0
    Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
    Date:   Tue Jun 29 13:43:09 2021 -0700

        lint fixes

    commit 6fff039fe122058837aefd621dd098592b761dde
    Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
    Date:   Mon Jun 28 16:44:38 2021 -0700

        yarn lint fix, comment divider fix

    commit 65a00ee6f7e962c17df9ef07b4d1bb0e4b08f1b9
    Author: Kaede Ito <kaede.ito2001@gmail.com>
    Date:   Mon Jun 28 15:09:34 2021 -0700

        Update .github/workflows/tests.yml

        Update wording

        Co-authored-by: Aidin Niavarani <aidin@wynnset.com>

    commit bc0c2eab93e924268ebea909720beb4532760fda
    Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
    Date:   Mon Jun 28 15:07:12 2021 -0700

        fixed typescript express.json() typing error

    commit dfb46417b5399176566d29a9db81f5c9f6547cc0
    Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
    Date:   Mon Jun 28 14:24:17 2021 -0700

        added pass & username to .env, small fix for tsconfig

    commit df1da6b1cc906509468010e75ce7e26d3dc698af
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Mon Jun 21 16:12:00 2021 -0700

        clean up

    commit c90970f5a01a95de6edd1841fda42309fefeff37
    Merge: 6dd0076 e961a61
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Mon Jun 21 16:07:48 2021 -0700

        Merge remote-tracking branch 'origin/neogma-tweaks' into neogma-fixtures

    commit e961a61895e7f475595ef7ea3a705e9b43f1f74c
    Merge: 621d318 a6ee549
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Mon Jun 21 16:04:40 2021 -0700

        Merge remote-tracking branch 'origin/update-docker-usage' into neogma-tweaks

    commit a6ee54907a01bb3c725ad18d39cb04d837e53919
    Merge: 7cddeae 8fb6807
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Mon Jun 21 16:03:23 2021 -0700

        Merge remote-tracking branch 'origin/model-schema-change' into update-docker-usage

    commit 6dd0076f3f10ee66730b661f7ca16433dec42112
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jun 21 15:27:26 2021 -0700

        small type fixes, remove unneeded imports

    commit 8fb6807accc204ba0a4d2a515b9c5b2b073645aa
    Merge: ed9adbd bb4d1c1
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jun 21 15:08:34 2021 -0700

        Merge branch 'neogma-installation' into model-schema-change

    commit 84c73bd90144d0600646890547a7d03539ecce2b
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Mon Jun 21 00:05:47 2021 -0700

        small refactor:

    commit 1b7956a0d066c2b6f1fdc5a87645eae228efedea
    Merge: de9b1fc 621d318
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Sun Jun 20 23:21:07 2021 -0700

        Merge remote-tracking branch 'origin/neogma-tweaks' into neogma-fixtures

    commit bb4d1c1569dbb0c815a9acedcb61f04009643b56
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Sun Jun 20 23:12:25 2021 -0700

        restore createUser

    commit f558fc8fd1c62562a1467718df9e49a8ca68f19e
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Sun Jun 20 23:11:14 2021 -0700

        restore old method names

    commit de9b1fcc36f3fa026249bdfa3b3125580d6054cf
    Merge: b5c267f 58a2337
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 22:09:49 2021 -0700

        Merge branch 'neogma-fixtures' of https://github.com/bipolarbridges/data-services into neogma-fixtures

    commit b5c267f91609c5355b4b36344594a0e77cc3f4c9
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 21:56:39 2021 -0700

        restore package.json

    commit 29b69760f505ab42bb6384d27e097613c06c2ea7
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 20:27:47 2021 -0700

        added examples fixtures and test data

    commit 7ff33c2f8bb36d97c72856ced364daaa621398a5
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 20:42:15 2021 -0700

        new testing workflow

    commit 7d9092e8d4c773eeaf3b7459397d026f23a0f154
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 18:33:45 2021 -0700

        added test data fixtures using neogma

    commit 58a2337f54a6593731f5245210cd8b6bdfdf10aa
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 22:04:13 2021 -0700

        added forgotten ;

    commit ed9adbd882eaa107961187d0e916f3d808e3c478
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 22:03:14 2021 -0700

        added catcher, added small ; fix

    commit 1dd9b1b5999c76e8d2c3aac48fc1e7e58da793bd
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 21:56:39 2021 -0700

        restore package.json

    commit d31f397a9ecd1bfa9287c0b9be8b5146cd00c8d5
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 20:27:47 2021 -0700

        added examples fixtures and test data

    commit a855608fecf12f1e94a6258de47f46cb3fb03f08
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 20:42:15 2021 -0700

        new testing workflow

    commit 9ded522b67cb6515dfb87d63e140636e809b5739
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 18:33:45 2021 -0700

        added test data fixtures using neogma

    commit 068d5094d7c23265bca18d7eee29776fbc3812c9
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 20:26:00 2021 -0700

        added measurement helper

    commit 4938b1bc02702c020c1c2e0523632567546c72c7
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 20:24:50 2021 -0700

        renamed from measurement value to measurement

    commit 621d31832c69275250407066c91f65222c79ab4b
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Fri Jun 18 17:06:28 2021 -0700

        fix the lint

    commit ea5d8b76bbfbe0cfc5e5f6d84e582f916c713e87
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Fri Jun 18 17:03:36 2021 -0700

        refactor existing methods to use OGM

    commit 7cddeae1d4b1e7b0a50900deed317a6b474234b5
    Merge: a2c1ad4 f6d4f5e
    Author: Kaede Ito <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 15:14:27 2021 -0700

        Merge branch 'model-schema-change' into update-docker-usage

    commit f6d4f5ede6ee53061838e474974a226448f00978
    Merge: 53a05b3 c7a4b31
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 15:09:23 2021 -0700

        Merge branch 'model-schema-change' of https://github.com/bipolarbridges/data-services into model-schema-change

    commit 53a05b3db6689017786cb92be960ed9f3703566a
    Merge: 214eb19 3049c5b
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 15:09:12 2021 -0700

        Merge branch 'neogma-installation' into model-schema-change

    commit 3049c5b6108672c4417e9f2812f8d92d21887564
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 14:43:55 2021 -0700

        made check for exising in endpoint

    commit 016f37874e73fafcc5e51c6014f8cf2e19e99bad
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 14:43:36 2021 -0700

        fixed duplicate relationship issue

    commit 65651ea7564a2315c6be514583451b7cb6830722
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 14:43:16 2021 -0700

        no space in recordedby

    commit 8bc5cd165928a8a2e04e0e49b9a33311df232099
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri Jun 18 14:06:53 2021 -0700

        added s

    commit 826bec71ab421f7dbfe038e4d2bd293fa22753a3
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Fri Jun 18 12:04:49 2021 -0700

        upgrade neogma

    commit 3e58f31dd492c189d1575c6e6fb375ff29949cee
    Merge: d479e04 a2c1ad4
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Fri Jun 18 11:39:44 2021 -0700

        Merge remote-tracking branch 'origin/update-docker-usage' into role-based-auth

    commit a2c1ad4c4223b974688452b4998f8423b8c76bfb
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Fri Jun 18 11:20:34 2021 -0700

        update network name usage

    commit c0aa2a19a3e530d994eb8abddc660f832f499d9d
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Fri Jun 18 11:18:44 2021 -0700

        use new docker compose syntax

    commit c7a4b3103420c06483133434ad8bb1ecfc54f703
    Merge: 214eb19 7f71c6d
    Author: Kaede Ito <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 19:50:36 2021 -0700

        Merge branch 'neogma-installation' into model-schema-change

    commit 7f71c6d57d4d917c7eb837c194fbac1bbdaeac99
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 19:38:09 2021 -0700

        adding build stage

    commit 214eb19bf6816888331bf1163e876123e1f39c03
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 18:50:54 2021 -0700

        reverted double relationship

    commit 8c3676a57dd569729ed146c86c061c177c03fec1
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 18:49:02 2021 -0700

        fixed typo

    commit 73f4a9ba4b6cdab31f4cf54135bd50666882cc8d
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 18:47:09 2021 -0700

        Reverted the change of modelrelatednode, added a note to the readme

    commit e1ddaa7012e245da64b44461f9e23f7003d19c6d
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 18:05:18 2021 -0700

        removed unused import

    commit fe36d8fd77593ab6f323ba81c11678d2a6c1d13d
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 18:04:13 2021 -0700

        added readme section on how to add more models

    commit 2778b2c7f648b8c2be6c7fa7a20757e45cf0abb9
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 18:04:01 2021 -0700

        moved all allModels type

    commit 368398ed0406cbc580ee47084a015ec11373033d
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 17:45:04 2021 -0700

        renamed controllers, made relatednode lowercase

    commit b1238a4b44e1b15144ce7174646e08df864c3f99
    Merge: def3572 3ffae04
    Author: Kaede Ito <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 16:42:29 2021 -0700

        Merge pull request #10 from bipolarbridges/qol-survey

        Merging QoL Data Export to simplify review.

    commit d479e04e052a5cba402da03fd680a27632ed9edd
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Wed Jun 16 13:36:42 2021 -0700

        typing tweaks

    commit 3ffae04a45dff3b2bbe26262fe0a3fe830d1e603
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 03:31:39 2021 -0700

        do undefined check before deconstruct

    commit e598ac7d28e720b407c3dc0d17d1ab218ff0417f
    Merge: 891e10a def3572
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 03:25:14 2021 -0700

        fixing merge conflicts

    commit def3572ade2067fb8643ad7b15da5a0f026343c5
    Merge: ddedd55 22259c8
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 03:22:07 2021 -0700

        Merge branch 'neogma-installation' into model-schema-change

    commit ddedd5564824f872271f396b0f299faec20319a8
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 03:20:38 2021 -0700

        fixed some tsc issues

    commit d95bacfd9d85b0630480cfd0ec2f81d17a57d67c
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 03:13:47 2021 -0700

        match types

    commit 891e10a7c89eee2dc309faf3fba99b450c6a640d
    Merge: 6d0a484 22259c8
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 03:00:20 2021 -0700

        Merge branch 'neogma-installation' into qol-survey

    commit 22259c85519f8ce81c65329e89c4d16f4ec6c4da
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 01:32:05 2021 -0700

        added parameter type, fixed type imports

    commit 2089bb1caa72c2c308f0bddb0b148126507513bc
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 01:29:57 2021 -0700

        skip spec.ts files

    commit 68c938be67f706acc61fa86d39a1dc933c206a2c
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 01:29:45 2021 -0700

        neogma has type error, skip module check

    commit 2934a07a688fa515f881d646053f0faa30c81929
    Merge: 5a0bc7a ab62e5e
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 00:45:51 2021 -0700

        Merge branch 'neogma-installation' of https://github.com/bipolarbridges/data-services into neogma-installation

    commit 5a0bc7a62362f04cba183ed8a8e79bfd3d50c3a7
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Wed Jun 16 00:45:45 2021 -0700

        linted, added types, removed unused functions

    commit 6d0a484d5c6ae414bebd84cfd721e252ad36ca7c
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Tue Jun 15 18:09:34 2021 -0700

        check  exist

    commit ab62e5e98b18be82fb39bfcb99f0fa78ef3a604b
    Merge: 4a078c6 aa2e305
    Author: rlnsy <rowlindsay@gmail.com>
    Date:   Mon Jun 14 23:06:16 2021 -0700

        Merge remote-tracking branch 'origin/master' into neogma-installation

    commit a7118dfa301586e640028e1c9128289cc0b97a81
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jun 14 14:25:58 2021 -0700

        changed date model

    commit 7353b5a6dd02d388a4dd241a82a821b810d8b501
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Jun 10 18:03:01 2021 -0700

        separated out into own const, use related to value now

    commit d206ba3c8cd9a02f57e64dca63a5948b3b86e638
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Jun 10 18:02:13 2021 -0700

        update with model

    commit d97dc160eef9be4abf86b52e3fb15512dec51782
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Jun 10 18:01:59 2021 -0700

        uses nodemon for lazy

    commit 1b43b7eef856c6407e917970ef8073cabcbbfb15
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Jun 10 18:01:40 2021 -0700

        renamed subtype into name

    commit eb42267391d1eab04c449997bb120ec481ffb7d1
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Jun 10 18:00:51 2021 -0700

        changed model name

    commit 97fbe460b30bb840ba40647c2379593a70513f75
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Tue Jun 8 18:06:36 2021 -0700

        model clean up, and adding building stage to pipeline

    commit 39be7c1161565ace7f1932c081e8eb64379e1e9a
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Tue Jun 8 16:47:57 2021 -0700

        fixed doubling on node making

    commit febdd45f5d0d2c620b82aa24ef3fbefd9338a311
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Tue Jun 8 15:58:43 2021 -0700

        reverted time to number, added user relationship to measurement

    commit 8be7499395c9d6b092cc94db4abee6aa3d640fd0
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jun 7 17:59:01 2021 -0700

        forgot to add measurement router

    commit 6c7edae377114585ca730df9503d9ec619e9d54b
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jun 7 17:48:55 2021 -0700

        modified to measurement creator to reflect new model

    commit 9864549cb103ccd024e994c45b684601a8f6bce1
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jun 7 17:48:24 2021 -0700

        modified endpoint to check more data keys

    commit 253ba07e0ae0bf1c530dc1a5d7114b8e4e525bd8
    Merge: 50c0fad 5649195
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon Jun 7 14:44:10 2021 -0700

        Merge branch 'model-schema-change' into qol-survey

    commit 564919592760253c150d2a4a8cbf7d05f5699a6e
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Jun 3 16:33:10 2021 -0700

        added resouce back to user

    commit 06a729494274e3d494f28a408d8111ba825e2553
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Jun 3 15:31:37 2021 -0700

        removed getting resource as client id

    commit b47f268e1cd9b8f611673b6c99551d5e6c20bf0b
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Jun 3 15:11:41 2021 -0700

        refactored to add new models

    commit 4a078c654749f7d93faaebb720b51eb84da947c7
    Merge: 1f9c903 c96e210
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu Jun 3 12:28:29 2021 -0700

        Merge branch 'typescript' into neogma-installation

    commit 1f9c903a3f9ec9ed9df2e068a94f77c7eac6ac76
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Tue Jun 1 19:55:26 2021 -0700

        hyphen is not allowed

    commit 6cecb9e1c98f875788ee476f8837260030559789
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Tue Jun 1 19:50:18 2021 -0700

        added hyphen to recorded-at

    commit bf4b52685b191f8e135b46ba0b5e4cfd52843e21
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Tue Jun 1 19:44:49 2021 -0700

        added missing properties

    commit 6fea05c0ab6ffb89d1773602ea1400026af9f3db
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Tue Jun 1 19:26:38 2021 -0700

        use merge, added initializers

    commit 1d0141bfc9886ef0ba8fa0cd96d6105cfcaa7eea
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Tue Jun 1 12:15:50 2021 -0700

        added catcher

    commit 73888dde68d1c1ee7efb5710b8a1e00153cec82b
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Tue Jun 1 11:57:10 2021 -0700

        added fix flagged lint

    commit aef938884c415e63817a9588ac0655bc9fda7c6d
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon May 31 18:27:25 2021 -0700

        added more controllers

    commit 040746a4a53642bb1a283d3f1ec8849d30701aea
    Merge: b3b3d46 1ac1953
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon May 31 15:45:54 2021 -0700

        Merge branch 'typescript' into neogma-installation

    commit b3b3d462218442f6a59f9035f79685773d898fa0
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Mon May 31 13:38:39 2021 -0700

        fleshed out relationships, finished createUser api controller

    commit 50c0fad4374d9e44c4c06a9ff67ff659358072c3
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Fri May 28 15:53:33 2021 -0700

        added helper function for date format

    commit 53341c8b14737e3729a611eb184dd44749ff9094
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu May 27 16:21:27 2021 -0700

        fixed import issue

    commit f904b1f107f6155dd9b2aeaaab9d3de5daa7743a
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu May 27 15:59:27 2021 -0700

        added resource

    commit a58f186258bc656b12c6f87dc382399281a231ee
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu May 27 15:43:30 2021 -0700

        fixed test, renamed label of Client to User

    commit 1dd95c5b0e1b19355283d25b615231211c229147
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu May 27 14:54:48 2021 -0700

        fixed import issue

    commit 9519e7f04bf0a737da42c55e4cb9b703fe7ca540
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Thu May 27 14:33:45 2021 -0700

        model creation

    commit 1cf5bfd9a39805be7c6c50acb7436aec0289f585
    Author: kaede <kaede.ito2001@gmail.com>
    Date:   Sat May 22 16:32:57 2021 -0700

        new paths for auth

    commit 8be852c4fa5a6f274beedaa307bcac765ab071fe
    Author: Kaede <kaede.ito2001@gmail.com>
    Date:   Thu May 20 15:33:07 2021 -0700

        added a createSurvey stub

    commit 449910a9f39501cf8fd036f13a4ba5e94db4e2aa
    Author: Kaede <kaede.ito2001@gmail.com>
    Date:   Thu May 20 14:21:31 2021 -0700

        added survey endpoint

commit 9915c9037e0144b0d3234ed842b9434f0b9ddb61
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jul 13 13:48:22 2021 -0700

    fixed import errors

commit 7c04c986064cc804049393bee04c1edda7fb4105
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jul 13 13:28:04 2021 -0700

    added health data case

commit c951ed88d2e8e8169c406cd2fc2539933f51b675
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jul 13 13:18:54 2021 -0700

    added path reading for ts-node

commit 155ab71cc775d24739c0a04f78a0c4376e96fefc
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jul 13 13:03:11 2021 -0700

    cleaner valid example

commit 868cd188e9fa0c290bee361320d83615c2182a15
Merge: e921831 02d31c1
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jul 12 18:08:09 2021 -0700

    Merge branch 'neogma-fixtures' into add-test-cases

commit 02d31c1468c1537d779e6297a492a336f2429989
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jul 5 18:02:05 2021 -0700

    lint and build fix

commit 1fb203cb568199602ddb254eb0a08a51b5ad534b
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jul 5 17:59:00 2021 -0700

    lint fix

commit 0a1c5858a6fabf6d48fd030f99c15111b9f42d6b
Merge: 3de979e df1da6b
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jul 5 17:56:43 2021 -0700

    Merge branch 'neogma-fixtures' of https://github.com/bipolarbridges/data-services into neogma-fixtures from rebasing

commit 3de979ef96f92d0bbb8c87bac2aee1f3862d7d28
Author: rlnsy <rowlindsay@gmail.com>
Date:   Mon Jun 21 16:12:00 2021 -0700

    clean up

commit 3e9752c2bbdeaf32033bcfbd34efcf03c6a49a86
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jun 21 15:27:26 2021 -0700

    small type fixes, remove unneeded imports

commit 16c4c93242f418666188560ebe4840f42a0cad24
Author: rlnsy <rowlindsay@gmail.com>
Date:   Mon Jun 21 00:05:47 2021 -0700

    small refactor:

commit 443ad138da7c722919ffd84acd728f0707468f07
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 18:33:45 2021 -0700

    added test data fixtures using neogma

commit b94e96f540e40791731e9e66e4749c3af66989ac
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 20:27:47 2021 -0700

    added examples fixtures and test data

commit e366ea5334da032b44873c7ef393d025c7d411f0
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 20:42:15 2021 -0700

    new testing workflow

commit 18654501803948e230c8a6df467c33f707f55309
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 18:33:45 2021 -0700

    added test data fixtures using neogma

commit 45f7bc400f52f842ea095f73f36206c8c33868a0
Merge: 38e0bd3 e961a61
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jul 5 17:41:28 2021 -0700

    Merge branch 'neogma-tweaks' of https://github.com/bipolarbridges/data-services into neogma-tweaks from rebasing

commit 38e0bd3b9f241699aec2fda58f43175b3b49ed1a
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jul 5 17:36:15 2021 -0700

    typing fix

commit b1159a695fc2e294e5d52f1889f47e606bff9226
Author: rlnsy <rowlindsay@gmail.com>
Date:   Fri Jun 18 17:06:28 2021 -0700

    fix the lint

commit a4da0fe7536a2f634e7d888bcd4561a4df0e5d5c
Author: rlnsy <rowlindsay@gmail.com>
Date:   Fri Jun 18 17:03:36 2021 -0700

    refactor existing methods to use OGM

commit 59f15cd6f2e2b67b2e4c1b8c4b9912e27224ba82
Author: rlnsy <rowlindsay@gmail.com>
Date:   Fri Jun 18 12:04:49 2021 -0700

    upgrade neogma

commit cf78305ecb56cc6fba980a7da08f0ed7e652daa0
Author: rlnsy <rowlindsay@gmail.com>
Date:   Wed Jun 16 13:36:42 2021 -0700

    typing tweaks

commit 1b3d3e0fe461ff39ca547949423f5cbed2d37cd8
Merge: 9390813 a6ee549
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jul 5 17:16:06 2021 -0700

    Merge branch 'update-docker-usage' of https://github.com/bipolarbridges/data-services into update-docker-usage

commit 9390813e93db2097493d1670571cfaede7407b5d
Author: rlnsy <rowlindsay@gmail.com>
Date:   Fri Jun 18 11:20:34 2021 -0700

    update network name usage

commit 42c41525db781a6ee67d1fc50daf0777d6326ea7
Author: rlnsy <rowlindsay@gmail.com>
Date:   Fri Jun 18 11:18:44 2021 -0700

    use new docker compose syntax

commit e92183143d7ab67e9866cadedfcfcdb83f762688
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jul 5 17:09:34 2021 -0700

    added more test cases

commit 905fc4f078cca05d28788c29e912b68764b92531
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 18:15:12 2021 -0700

    cleaned up comments, made initializer live in model files

commit 705a7bcc40edfb41e6ceee4c3bd6885b598bea6d
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 17:50:27 2021 -0700

    readme change

commit 4156f3157fc06a1df18d08f1bee7fab57e1b9208
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 17:37:54 2021 -0700

    dependency cycle fix

commit c62e43fd9ca13c00eb853cc3411d6fa83db30542
Merge: 768df79 955ccc7
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 17:32:57 2021 -0700

    Merge branch 'model-schema-change' into fix

commit 768df791bf13907e0da0d95a581690c2edf4fbec
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 17:30:11 2021 -0700

    lint fix

commit 7ab46d9f21b2104cb32d44294df4efc93e890a34
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 16:46:30 2021 -0700

    lint:fix

commit 3bcadb25911fc82b4461372b872883addf92cb6f
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 16:35:27 2021 -0700

    more dependencies

commit 99914db8a26e1bd68a52bce4e56fbf809549ada1
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 16:34:42 2021 -0700

    eslint fix

commit b79f4f9f5a1bda32df3f628aa1cd1589b38ea445
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 16:34:25 2021 -0700

    yarn lock

commit 955ccc71ea05f9354817bcb85aded4f1394a8268
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 16:08:59 2021 -0700

    no async

commit a12ef566aaf4b299bb5e5710069a65c3b754a31a
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 15:15:32 2021 -0700

    more lint fix, yarn lock fix

commit 0f2222c77441331b141882fac414d90f146c0b5e
Merge: 978bafb 109c2e1
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 15:08:57 2021 -0700

    Merge branch 'neogma-installation' into model-schema-change

commit 109c2e1ec48ce0148c515107ffc7efd87df40642
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 14:56:35 2021 -0700

    lint fix

commit 1bf5a760695e6b37bc9d1c87558d3f1474fae023
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 13:48:36 2021 -0700

    adding missing pckgs

commit 682645325af8188362f9934d65c13bd95fcb78fe
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 30 13:46:26 2021 -0700

    linter change

commit 978bafbbf1c51137136a08e65dea03fb6b95f578
Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
Date:   Tue Jun 29 17:18:16 2021 -0700

    removed addRelationship steps

commit 8d3aa96f14cd0ef0bc70a3c17193290015af7b4f
Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
Date:   Tue Jun 29 16:41:32 2021 -0700

    lint fix

commit c6fe1aba8a53bd56fc4556c852a00492b8590c13
Merge: 9ae1492 bc0c2ea
Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
Date:   Tue Jun 29 14:30:41 2021 -0700

    Merge branch 'neogma-installation' into model-schema-change

commit 9ae149281b95b343a8ff0d7e745fbde004a021df
Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
Date:   Tue Jun 29 14:16:38 2021 -0700

    removed unnecessary comments

commit b9a6b3ed3ae9a252126d449f2399256853089747
Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
Date:   Tue Jun 29 14:11:29 2021 -0700

    type fix

commit d14d5e29a07a92cc5ea19fba11384f2ff4270e64
Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
Date:   Tue Jun 29 14:07:05 2021 -0700

    circular dependency fix

commit 24914287391b6eb2a4a80f421d946dde75b702b0
Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
Date:   Tue Jun 29 13:43:09 2021 -0700

    lint fixes

commit 6fff039fe122058837aefd621dd098592b761dde
Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
Date:   Mon Jun 28 16:44:38 2021 -0700

    yarn lint fix, comment divider fix

commit 65a00ee6f7e962c17df9ef07b4d1bb0e4b08f1b9
Author: Kaede Ito <kaede.ito2001@gmail.com>
Date:   Mon Jun 28 15:09:34 2021 -0700

    Update .github/workflows/tests.yml

    Update wording

    Co-authored-by: Aidin Niavarani <aidin@wynnset.com>

commit bc0c2eab93e924268ebea909720beb4532760fda
Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
Date:   Mon Jun 28 15:07:12 2021 -0700

    fixed typescript express.json() typing error

commit dfb46417b5399176566d29a9db81f5c9f6547cc0
Author: Mapleia <36678302+Mapleia@users.noreply.github.com>
Date:   Mon Jun 28 14:24:17 2021 -0700

    added pass & username to .env, small fix for tsconfig

commit 427ea59651f7f49104907c91096b4ad44bfe2bba
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jun 21 18:19:46 2021 -0700

    new test cases

commit df1da6b1cc906509468010e75ce7e26d3dc698af
Author: rlnsy <rowlindsay@gmail.com>
Date:   Mon Jun 21 16:12:00 2021 -0700

    clean up

commit c90970f5a01a95de6edd1841fda42309fefeff37
Merge: 6dd0076 e961a61
Author: rlnsy <rowlindsay@gmail.com>
Date:   Mon Jun 21 16:07:48 2021 -0700

    Merge remote-tracking branch 'origin/neogma-tweaks' into neogma-fixtures

commit e961a61895e7f475595ef7ea3a705e9b43f1f74c
Merge: 621d318 a6ee549
Author: rlnsy <rowlindsay@gmail.com>
Date:   Mon Jun 21 16:04:40 2021 -0700

    Merge remote-tracking branch 'origin/update-docker-usage' into neogma-tweaks

commit a6ee54907a01bb3c725ad18d39cb04d837e53919
Merge: 7cddeae 8fb6807
Author: rlnsy <rowlindsay@gmail.com>
Date:   Mon Jun 21 16:03:23 2021 -0700

    Merge remote-tracking branch 'origin/model-schema-change' into update-docker-usage

commit 6dd0076f3f10ee66730b661f7ca16433dec42112
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jun 21 15:27:26 2021 -0700

    small type fixes, remove unneeded imports

commit 8fb6807accc204ba0a4d2a515b9c5b2b073645aa
Merge: ed9adbd bb4d1c1
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jun 21 15:08:34 2021 -0700

    Merge branch 'neogma-installation' into model-schema-change

commit 84c73bd90144d0600646890547a7d03539ecce2b
Author: rlnsy <rowlindsay@gmail.com>
Date:   Mon Jun 21 00:05:47 2021 -0700

    small refactor:

commit 1b7956a0d066c2b6f1fdc5a87645eae228efedea
Merge: de9b1fc 621d318
Author: rlnsy <rowlindsay@gmail.com>
Date:   Sun Jun 20 23:21:07 2021 -0700

    Merge remote-tracking branch 'origin/neogma-tweaks' into neogma-fixtures

commit bb4d1c1569dbb0c815a9acedcb61f04009643b56
Author: rlnsy <rowlindsay@gmail.com>
Date:   Sun Jun 20 23:12:25 2021 -0700

    restore createUser

commit f558fc8fd1c62562a1467718df9e49a8ca68f19e
Author: rlnsy <rowlindsay@gmail.com>
Date:   Sun Jun 20 23:11:14 2021 -0700

    restore old method names

commit de9b1fcc36f3fa026249bdfa3b3125580d6054cf
Merge: b5c267f 58a2337
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 22:09:49 2021 -0700

    Merge branch 'neogma-fixtures' of https://github.com/bipolarbridges/data-services into neogma-fixtures

commit b5c267f91609c5355b4b36344594a0e77cc3f4c9
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 21:56:39 2021 -0700

    restore package.json

commit 29b69760f505ab42bb6384d27e097613c06c2ea7
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 20:27:47 2021 -0700

    added examples fixtures and test data

commit 7ff33c2f8bb36d97c72856ced364daaa621398a5
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 20:42:15 2021 -0700

    new testing workflow

commit 7d9092e8d4c773eeaf3b7459397d026f23a0f154
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 18:33:45 2021 -0700

    added test data fixtures using neogma

commit 58a2337f54a6593731f5245210cd8b6bdfdf10aa
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 22:04:13 2021 -0700

    added forgotten ;

commit ed9adbd882eaa107961187d0e916f3d808e3c478
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 22:03:14 2021 -0700

    added catcher, added small ; fix

commit 1dd9b1b5999c76e8d2c3aac48fc1e7e58da793bd
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 21:56:39 2021 -0700

    restore package.json

commit d31f397a9ecd1bfa9287c0b9be8b5146cd00c8d5
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 20:27:47 2021 -0700

    added examples fixtures and test data

commit a855608fecf12f1e94a6258de47f46cb3fb03f08
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 20:42:15 2021 -0700

    new testing workflow

commit 9ded522b67cb6515dfb87d63e140636e809b5739
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 18:33:45 2021 -0700

    added test data fixtures using neogma

commit 068d5094d7c23265bca18d7eee29776fbc3812c9
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 20:26:00 2021 -0700

    added measurement helper

commit 4938b1bc02702c020c1c2e0523632567546c72c7
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 20:24:50 2021 -0700

    renamed from measurement value to measurement

commit 621d31832c69275250407066c91f65222c79ab4b
Author: rlnsy <rowlindsay@gmail.com>
Date:   Fri Jun 18 17:06:28 2021 -0700

    fix the lint

commit ea5d8b76bbfbe0cfc5e5f6d84e582f916c713e87
Author: rlnsy <rowlindsay@gmail.com>
Date:   Fri Jun 18 17:03:36 2021 -0700

    refactor existing methods to use OGM

commit 7cddeae1d4b1e7b0a50900deed317a6b474234b5
Merge: a2c1ad4 f6d4f5e
Author: Kaede Ito <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 15:14:27 2021 -0700

    Merge branch 'model-schema-change' into update-docker-usage

commit f6d4f5ede6ee53061838e474974a226448f00978
Merge: 53a05b3 c7a4b31
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 15:09:23 2021 -0700

    Merge branch 'model-schema-change' of https://github.com/bipolarbridges/data-services into model-schema-change

commit 53a05b3db6689017786cb92be960ed9f3703566a
Merge: 214eb19 3049c5b
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 15:09:12 2021 -0700

    Merge branch 'neogma-installation' into model-schema-change

commit 3049c5b6108672c4417e9f2812f8d92d21887564
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 14:43:55 2021 -0700

    made check for exising in endpoint

commit 016f37874e73fafcc5e51c6014f8cf2e19e99bad
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 14:43:36 2021 -0700

    fixed duplicate relationship issue

commit 65651ea7564a2315c6be514583451b7cb6830722
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 14:43:16 2021 -0700

    no space in recordedby

commit 8bc5cd165928a8a2e04e0e49b9a33311df232099
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri Jun 18 14:06:53 2021 -0700

    added s

commit 826bec71ab421f7dbfe038e4d2bd293fa22753a3
Author: rlnsy <rowlindsay@gmail.com>
Date:   Fri Jun 18 12:04:49 2021 -0700

    upgrade neogma

commit 3e58f31dd492c189d1575c6e6fb375ff29949cee
Merge: d479e04 a2c1ad4
Author: rlnsy <rowlindsay@gmail.com>
Date:   Fri Jun 18 11:39:44 2021 -0700

    Merge remote-tracking branch 'origin/update-docker-usage' into role-based-auth

commit a2c1ad4c4223b974688452b4998f8423b8c76bfb
Author: rlnsy <rowlindsay@gmail.com>
Date:   Fri Jun 18 11:20:34 2021 -0700

    update network name usage

commit c0aa2a19a3e530d994eb8abddc660f832f499d9d
Author: rlnsy <rowlindsay@gmail.com>
Date:   Fri Jun 18 11:18:44 2021 -0700

    use new docker compose syntax

commit c7a4b3103420c06483133434ad8bb1ecfc54f703
Merge: 214eb19 7f71c6d
Author: Kaede Ito <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 19:50:36 2021 -0700

    Merge branch 'neogma-installation' into model-schema-change

commit 7f71c6d57d4d917c7eb837c194fbac1bbdaeac99
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 19:38:09 2021 -0700

    adding build stage

commit 214eb19bf6816888331bf1163e876123e1f39c03
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 18:50:54 2021 -0700

    reverted double relationship

commit 8c3676a57dd569729ed146c86c061c177c03fec1
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 18:49:02 2021 -0700

    fixed typo

commit 73f4a9ba4b6cdab31f4cf54135bd50666882cc8d
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 18:47:09 2021 -0700

    Reverted the change of modelrelatednode, added a note to the readme

commit e1ddaa7012e245da64b44461f9e23f7003d19c6d
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 18:05:18 2021 -0700

    removed unused import

commit fe36d8fd77593ab6f323ba81c11678d2a6c1d13d
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 18:04:13 2021 -0700

    added readme section on how to add more models

commit 2778b2c7f648b8c2be6c7fa7a20757e45cf0abb9
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 18:04:01 2021 -0700

    moved all allModels type

commit 368398ed0406cbc580ee47084a015ec11373033d
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 17:45:04 2021 -0700

    renamed controllers, made relatednode lowercase

commit b1238a4b44e1b15144ce7174646e08df864c3f99
Merge: def3572 3ffae04
Author: Kaede Ito <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 16:42:29 2021 -0700

    Merge pull request #10 from bipolarbridges/qol-survey

    Merging QoL Data Export to simplify review.

commit d479e04e052a5cba402da03fd680a27632ed9edd
Author: rlnsy <rowlindsay@gmail.com>
Date:   Wed Jun 16 13:36:42 2021 -0700

    typing tweaks

commit 3ffae04a45dff3b2bbe26262fe0a3fe830d1e603
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 03:31:39 2021 -0700

    do undefined check before deconstruct

commit e598ac7d28e720b407c3dc0d17d1ab218ff0417f
Merge: 891e10a def3572
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 03:25:14 2021 -0700

    fixing merge conflicts

commit def3572ade2067fb8643ad7b15da5a0f026343c5
Merge: ddedd55 22259c8
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 03:22:07 2021 -0700

    Merge branch 'neogma-installation' into model-schema-change

commit ddedd5564824f872271f396b0f299faec20319a8
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 03:20:38 2021 -0700

    fixed some tsc issues

commit d95bacfd9d85b0630480cfd0ec2f81d17a57d67c
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 03:13:47 2021 -0700

    match types

commit 891e10a7c89eee2dc309faf3fba99b450c6a640d
Merge: 6d0a484 22259c8
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 03:00:20 2021 -0700

    Merge branch 'neogma-installation' into qol-survey

commit 22259c85519f8ce81c65329e89c4d16f4ec6c4da
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 01:32:05 2021 -0700

    added parameter type, fixed type imports

commit 2089bb1caa72c2c308f0bddb0b148126507513bc
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 01:29:57 2021 -0700

    skip spec.ts files

commit 68c938be67f706acc61fa86d39a1dc933c206a2c
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 01:29:45 2021 -0700

    neogma has type error, skip module check

commit 2934a07a688fa515f881d646053f0faa30c81929
Merge: 5a0bc7a ab62e5e
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 00:45:51 2021 -0700

    Merge branch 'neogma-installation' of https://github.com/bipolarbridges/data-services into neogma-installation

commit 5a0bc7a62362f04cba183ed8a8e79bfd3d50c3a7
Author: kaede <kaede.ito2001@gmail.com>
Date:   Wed Jun 16 00:45:45 2021 -0700

    linted, added types, removed unused functions

commit 6d0a484d5c6ae414bebd84cfd721e252ad36ca7c
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jun 15 18:09:34 2021 -0700

    check  exist

commit ab62e5e98b18be82fb39bfcb99f0fa78ef3a604b
Merge: 4a078c6 aa2e305
Author: rlnsy <rowlindsay@gmail.com>
Date:   Mon Jun 14 23:06:16 2021 -0700

    Merge remote-tracking branch 'origin/master' into neogma-installation

commit a7118dfa301586e640028e1c9128289cc0b97a81
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jun 14 14:25:58 2021 -0700

    changed date model

commit 7353b5a6dd02d388a4dd241a82a821b810d8b501
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu Jun 10 18:03:01 2021 -0700

    separated out into own const, use related to value now

commit d206ba3c8cd9a02f57e64dca63a5948b3b86e638
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu Jun 10 18:02:13 2021 -0700

    update with model

commit d97dc160eef9be4abf86b52e3fb15512dec51782
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu Jun 10 18:01:59 2021 -0700

    uses nodemon for lazy

commit 1b43b7eef856c6407e917970ef8073cabcbbfb15
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu Jun 10 18:01:40 2021 -0700

    renamed subtype into name

commit eb42267391d1eab04c449997bb120ec481ffb7d1
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu Jun 10 18:00:51 2021 -0700

    changed model name

commit 97fbe460b30bb840ba40647c2379593a70513f75
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jun 8 18:06:36 2021 -0700

    model clean up, and adding building stage to pipeline

commit 39be7c1161565ace7f1932c081e8eb64379e1e9a
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jun 8 16:47:57 2021 -0700

    fixed doubling on node making

commit febdd45f5d0d2c620b82aa24ef3fbefd9338a311
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jun 8 15:58:43 2021 -0700

    reverted time to number, added user relationship to measurement

commit 8be7499395c9d6b092cc94db4abee6aa3d640fd0
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jun 7 17:59:01 2021 -0700

    forgot to add measurement router

commit 6c7edae377114585ca730df9503d9ec619e9d54b
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jun 7 17:48:55 2021 -0700

    modified to measurement creator to reflect new model

commit 9864549cb103ccd024e994c45b684601a8f6bce1
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jun 7 17:48:24 2021 -0700

    modified endpoint to check more data keys

commit 253ba07e0ae0bf1c530dc1a5d7114b8e4e525bd8
Merge: 50c0fad 5649195
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon Jun 7 14:44:10 2021 -0700

    Merge branch 'model-schema-change' into qol-survey

commit 564919592760253c150d2a4a8cbf7d05f5699a6e
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu Jun 3 16:33:10 2021 -0700

    added resouce back to user

commit 06a729494274e3d494f28a408d8111ba825e2553
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu Jun 3 15:31:37 2021 -0700

    removed getting resource as client id

commit b47f268e1cd9b8f611673b6c99551d5e6c20bf0b
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu Jun 3 15:11:41 2021 -0700

    refactored to add new models

commit 4a078c654749f7d93faaebb720b51eb84da947c7
Merge: 1f9c903 c96e210
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu Jun 3 12:28:29 2021 -0700

    Merge branch 'typescript' into neogma-installation

commit 1f9c903a3f9ec9ed9df2e068a94f77c7eac6ac76
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jun 1 19:55:26 2021 -0700

    hyphen is not allowed

commit 6cecb9e1c98f875788ee476f8837260030559789
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jun 1 19:50:18 2021 -0700

    added hyphen to recorded-at

commit bf4b52685b191f8e135b46ba0b5e4cfd52843e21
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jun 1 19:44:49 2021 -0700

    added missing properties

commit 6fea05c0ab6ffb89d1773602ea1400026af9f3db
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jun 1 19:26:38 2021 -0700

    use merge, added initializers

commit 1d0141bfc9886ef0ba8fa0cd96d6105cfcaa7eea
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jun 1 12:15:50 2021 -0700

    added catcher

commit 73888dde68d1c1ee7efb5710b8a1e00153cec82b
Author: kaede <kaede.ito2001@gmail.com>
Date:   Tue Jun 1 11:57:10 2021 -0700

    added fix flagged lint

commit aef938884c415e63817a9588ac0655bc9fda7c6d
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon May 31 18:27:25 2021 -0700

    added more controllers

commit 040746a4a53642bb1a283d3f1ec8849d30701aea
Merge: b3b3d46 1ac1953
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon May 31 15:45:54 2021 -0700

    Merge branch 'typescript' into neogma-installation

commit b3b3d462218442f6a59f9035f79685773d898fa0
Author: kaede <kaede.ito2001@gmail.com>
Date:   Mon May 31 13:38:39 2021 -0700

    fleshed out relationships, finished createUser api controller

commit 50c0fad4374d9e44c4c06a9ff67ff659358072c3
Author: kaede <kaede.ito2001@gmail.com>
Date:   Fri May 28 15:53:33 2021 -0700

    added helper function for date format

commit 53341c8b14737e3729a611eb184dd44749ff9094
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu May 27 16:21:27 2021 -0700

    fixed import issue

commit f904b1f107f6155dd9b2aeaaab9d3de5daa7743a
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu May 27 15:59:27 2021 -0700

    added resource

commit a58f186258bc656b12c6f87dc382399281a231ee
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu May 27 15:43:30 2021 -0700

    fixed test, renamed label of Client to User

commit 1dd95c5b0e1b19355283d25b615231211c229147
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu May 27 14:54:48 2021 -0700

    fixed import issue

commit 9519e7f04bf0a737da42c55e4cb9b703fe7ca540
Author: kaede <kaede.ito2001@gmail.com>
Date:   Thu May 27 14:33:45 2021 -0700

    model creation

commit 1cf5bfd9a39805be7c6c50acb7436aec0289f585
Author: kaede <kaede.ito2001@gmail.com>
Date:   Sat May 22 16:32:57 2021 -0700

    new paths for auth

commit 8be852c4fa5a6f274beedaa307bcac765ab071fe
Author: Kaede <kaede.ito2001@gmail.com>
Date:   Thu May 20 15:33:07 2021 -0700

    added a createSurvey stub

commit 449910a9f39501cf8fd036f13a4ba5e94db4e2aa
Author: Kaede <kaede.ito2001@gmail.com>
Date:   Thu May 20 14:21:31 2021 -0700

    added survey endpoint

noktoa committed:
restructured api testing
